### PR TITLE
Sscs 5793 clear hmcts dwp state

### PR DESCRIPTION
**Pull request by jackmaloney**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-5793

### Change description ###
Clear hmctsDwpState field on sendToDwpOffline event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import uk.gov.hmcts.reform.sscs.ccd.deserialisation.SscsCaseCallbackDeserializer;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsDocument;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackDispatcher;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
+import uk.gov.hmcts.reform.sscs.controller.CcdCallbackController;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+import uk.gov.hmcts.reform.sscs.service.AuthorisationService;
+
+@RunWith(JUnitParamsRunner.class)
```
Comment (iguannago) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java]: it's not used.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+@SpringBootTest
+@ActiveProfiles("integration")
+@AutoConfigureMockMvc
+public class CcdCallbackEndpointIt {
+    // Below rules are needed to use the junitParamsRunner together with SpringRunner
+    @ClassRule
```
Comment (iguannago) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java]: it's not used.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    public static final SpringClassRule SPRING_CLASS_RULE = new SpringClassRule();
+
+    @Rule
+    public final SpringMethodRule springMethodRule = new SpringMethodRule();
```
Comment (iguannago) [inline src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java]: not needed.

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java (hunk 2 above)
Comment (jackmaloney) [reply src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java]: done

File: src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java (hunk 3 above)
Comment (jackmaloney) [reply src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/event/CcdCallbackEndpointIt.java]: tests fail when this is removed so keeping it in

**Pull request merged**